### DISCUSSION TITLE
How about to use errors (stdlib) instead of friendsofgo/errors?

### DISCUSSION THREAD

**Discussion by pankona** [aarondl/sqlboiler#1162] [Ideas]
It seems sqlboiler depends on https://github.com/friendsofgo/errors. I got it was introduced to support go 1.13's error handling (Refs: https://github.com/volatiletech/sqlboiler/pull/624).

Is there any chance to migrate it to errors package in stdlib? I think using errors package in stdlib is better than using friendsofgo/errors from the perspective of maintainability and sustainability. And also it contributes to reduce number of dependencies for the applications that uses sqlboiler.

I'm not sure there're blocker (or some breaking change) if sqlboiler uses errors package in stdlib but I'm happy I'd hear your opinion. Thanks!

**Comment by stephenafamo** [comment]
It will likely be a breaking change for anyone depending on `errors.Cause`.

Also, as an improvement to `fmt.Errorf`, using `errors.Wrap()` includes a stack trace. So some may prefer it.
